src/Locator/BundleResourceLocator.php

```diff
@@ -66,7 +63,7 @@ private function locateResourceBasedOnBundleNotation(string $resourcePath, Theme
 
         // Symfony 4.0+ always returns a single bundle
         /** @var BundleInterface|BundleInterface[] $bundles */
-        $bundles = $this->kernel->getBundle($bundleName, false);
+        $bundles = $this->kernel->getBundle($bundleName);
```

Comment:
Probably it should be refactored according to the comment above

Reply:
IMO its necessary because interface doesn't have second parameter 
![Screenshot 2020-02-20 at 14 57 52](https://user-images.githubusercontent.com/39232096/74940341-8f9ee380-53f1-11ea-8639-d8b0de4df472.png)

Reply:
`// Symfony 4.0+ always returns a single bundle` this is the comment that I wrote about